content/pytorch/concepts/tensors/terms/rand/rand.md

```diff
@@ -0,0 +1,106 @@
+---
+Title: 'rand()'
+Description: 'rand() function returns a desired size tensor filled with the random values that generated from uniform distribution in the range [0,1).'
+
+Subjects:
+  - 'AI'
+  - 'Data Science'
+  - 'Machine Learning'
+Tags:
+  - 'AI'
+  - 'Deep Learning'
+  - 'Machine Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+  - 'paths/data-science'
+  - 'learn-python-3'
+---
+
+**.rand()** .rand() is used to return a tensor of the desired size, filled with random numbers generated from a uniform distribution in the range [0, 1), where the endpoint value is not included (exclusive), and all numbers in this range have an equal probability of being chosen.
+
+## Syntax
+
+`.rand()` in PyTorch is part of the `torch` package and the syntax shown below:
+
+```pseudo
+torch.rand(*size, *, generator=None, out=None, dtype=None, layout=torch.strided, device=None requires_grad=False, pin_memory=False)
+
+```
+
+`torch.rand()` take one parameter `size` which represent the shape of the tensor that wish to create. It can create a tensor with one, two, or more dimensions.
+
+Meanwhile, `torch.rand()` have other seven optional arguments that can be used to customize the tensor if needed.
+
+- `generator` is used for generating number for sampling.
+- `out`, to store the output into existing tensor (if there is any).
+- `dtype` is used to customize the data type of the output.
+- `layout`, to specify how Tensor is stored in memory.
+- `device`, to specify where the tensor will be created.
+- `requires_grad`, to record the operation of the tensor.
+- `pin_memory` is used for allocating the tensor in the pinned memory.
```

Comment:
Modifications:

- Let's change the sentences a bit
- Let's convert commas after the parameters into colons
- Let's change the parameter descriptions a bit

```suggestion
The `.rand()` function takes one required parameter `size`, which represents the shape of the tensor that is to be created. It can create a tensor with one, two, or more dimensions.

Meanwhile, there are seven other optional parameters that can be used with it to customize the tensor if needed:

- `generator`: Used for generating a number for sampling.
- `out`: Stores the output into existing tensor (if there is any).
- `dtype`: Used to customize the data type of the output.
- `layout`: Specifies how the tensor is stored in the memory.
- `device`: Specifies where the tensor will be created.
- `requires_grad`: Records the operation of the tensor.
- `pin_memory`: Used for allocating the tensor in the pinned memory.
```